testdata/go/v3/memcached-operator/Makefile

```diff
@@ -88,7 +88,7 @@ vet: ## Run go vet against code.
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: manifests generate fmt vet ## Run tests.
 	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.2/hack/setup-envtest.sh
```

Comment:
can you rebase with master and generate the samples again? This needs to be reverted back since controller-runtime was bumped to 0.8.3

Reply:
This is a result of java-operator-plugins using an [older version of kubebuilder](https://github.com/operator-framework/java-operator-plugins/blob/main/go.mod#L11) and go modules' min version algorithm. operator-sdk needs to pin to a specific version of kubebuilder with `replace` to avoid this issue now and in the future when more plugins are imported.

Reply:
ah, okay..